builder/dockerfile/copy.go

```diff
@@ -68,10 +68,10 @@ type copier struct {
 	tmpPaths    []string
 }
 
-func copierFromDispatchRequest(req dispatchRequest, download sourceDownloader, imageSource *imageMount) copier {
+func newCopier(source builder.Source, cache pathCache, download sourceDownloader, imageSource *imageMount) copier {
```

Comment:
This is because it does not create a copier from a dispatchRequest anymore, but 2 components to reduce the necessary interface (although, it was initially done because at some point there was no dispatchRequest type anymore)